estimagic/optimization/optimize.py

```diff
@@ -156,51 +135,49 @@ def minimize(
     Args:
         criterion (function or list of functions):
             Python function that takes a pandas DataFrame with parameters as the first
-            argument and returns a scalar floating point value.
-
+            argument. Supported outputs are:
```

Comment:
@tobiasraabe can you clarify the supported outputs? I think the numpy array output is not only for pounders but rather for log likelihoods per observation but I am not sure.